network-store-server/src/main/resources/db/changelog/changesets/refactor_limits_20241121T110000Z.sql

```diff
@@ -0,0 +1,22 @@
+-- will be used in next deployment
```

Comment:
I was thinking of doing the liquibase migration by doing a java change that just calls the same method as the one used for users